### DISCUSSION TITLE
vlc (again)

### DISCUSSION THREAD

**Discussion by macerg** [termux/termux-packages#10242] [Q&A] [answered]
vlc has again been upgraded (to VLC media player 3.0.17.3 Vetinari (revision 3.0.13-8-g41878ff4f2)) but STILL doesn't include the qt interface (for x11). The termux support group say they don't have resources but have just upgraded but failed to include an interface that I have requested many times. Is it so difficult? Please discuss.

**Comment by ghost** [comment]
This depends on the complexity with inclusion of graphical interface. I'm not sure how vlc with gui works correctly when enabling GUI like video playback and support libraries. I'll try to look how hard it is to enable QT

And please be patient with your issue. we only have few people are working with this project and we are not paying them

**Reply by macerg** [reply]
Thanks.

**Comment by ghost** [comment, marked as answer]
VLC with QT support (`vlc-qt`) is now merged and should be available for few minutes and for a while for mirrors
